### PR TITLE
pr split 14 - add 10 files

### PR DESCRIPTION
**Pull request by michaelkernaghan**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have added unit tests
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/wallet-with-bad-code.spec.ts [hunk 1]
```diff
@@ -3,13 +3,13 @@ import { badCode } from "./data/badCode";
...
   const Tezos = lib;
-  describe(`Test contract made from wallet API failing with bad code using: ${rpc}`, () => {
+  describe(`Test contract origination fail with bad code through wallet api using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    it('fails with a 400 RPC response', async (done) => {
+    it('Verify that wallet.originate for a contract with 400 in code will return 400 RPC in response', async (done) => {
```
Comment (roxaneletourneau) [inline integration-tests/wallet-with-bad-code.spec.ts:12]: There is no 400 in the contract code, but an instruction that doesn't exist : FAILWITH_TYPO

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: integration-tests/wallet-register-delegate.spec.ts [hunk 2]
```diff
@@ -3,12 +3,12 @@ import { CONFIGS } from './config';
 
 CONFIGS().forEach(({ lib, rpc, setup, protocol }) => {
     const Tezos = lib;
-    describe(`Test  register delegate: ${rpc}`, () => {
+    describe(`Test address register delegate through wallet api: ${rpc}`, () => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/wallet-register-delegate.spec.ts:6]: changed to "`Test delegate registration through wallet api"

File: integration-tests/wallet-tx-wait-2-confirmations.spec.ts [hunk 3]
```diff
@@ -2,14 +2,14 @@ import { CONFIGS } from "./config";
...
-  describe(`Test tx and waiting for 2 confirmations using: ${rpc}`, () => {
+  describe(`Test tx and waiting for 2 confirmations through wallet api using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
-    it('transfers 0.02 tez and waits for 2 confirmations with wallet', async (done) => {
-      const op = await Tezos.wallet.transfer({ to: 'tz1ZfrERcALBwmAqwonRXYVQBDT9BjNjBHJu', amount: 0.02 }).send();
+    it('Verify contract.transfer for 2 XTZ to an address and waits for 2 confirmations', async (done) => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/wallet-tx-wait-2-confirmations.spec.ts:11]: fixed

File: integration-tests/wallet-unit-as-param.spec.ts [hunk 4]
```diff
@@ -3,14 +3,14 @@ import { depositContractCode, depositContractStorage } from "./data/deposit_cont
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
+  const test = require('jest-retries');
+  describe(`Test contract origination with unit as params through wallet api using: ${rpc}`, () => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/wallet-unit-as-param.spec.ts:7]: fixed

File: integration-tests/wallet-with-bad-code.spec.ts (hunk 1 above)
Comment (michaelkernaghan) [reply integration-tests/wallet-with-bad-code.spec.ts:12]: fixed

File: integration-tests/wallet-originate-vote-contract.spec.ts [hunk 5]
```diff
@@ -4,13 +4,13 @@ import { voteSample } from "./data/vote-contract";
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
   const test = require('jest-retries');
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/wallet-originate-vote-contract.spec.ts]: removed

File: integration-tests/wallet-origination-wrong-order-code-properties.spec.ts [hunk 6]
```diff
@@ -5,13 +5,13 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
...
 
-  describe(`Originate contract with code properties in atypical order : ${rpc}`, () => {
+  describe(`Test contract origination with code properties in atypical order through wallet api: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup()
       done()
     })
- test('originates a wallet api contract despite the code storage, parameter, and code props being in wrong order', 2, async (done: () => void) => {
+ test('Verify wallet.originate for a contract despite the code storage,parameter,code props are in wrong order', 2, async (done: () => void) => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/wallet-origination-wrong-order-code-properties.spec.ts]: fixed

File: integration-tests/wallet-register-delegate.spec.ts [hunk 7]
```diff
@@ -3,12 +3,12 @@ import { CONFIGS } from './config';
...
         beforeEach(async (done) => {
             await setup(true);
             done();
         });
-        it('registers the current address as delegate', async (done) => {
+        it('As a User I want to verify that I can register the current address as delegate using wallet.registerDelegate', async (done) => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/wallet-register-delegate.spec.ts]: change to Verify

File: integration-tests/wallet-set-delegate-auto-estimate.spec.ts [hunk 8]
```diff
@@ -2,13 +2,13 @@ import { CONFIGS } from "./config";
...
   const Tezos = lib;
-  describe(`Test account delegation with estimation using: ${rpc}`, () => {
+  describe(`Test account delegation with estimation through wallet api using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup(true)
       done()
     })
-    it('delegates account to known baker with automatic estimate', async (done) => {
+    it('As a User I want to verify that I can delegate account to known baker with automatic estimate', async (done) => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/wallet-set-delegate-auto-estimate.spec.ts]: changed

File: integration-tests/wallet-simple-origination.spec.ts [hunk 9]
```diff
@@ -2,13 +2,13 @@ import { CONFIGS } from "./config";
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
-  describe(`Originating a contract from wallet api using: ${rpc}`, () => {
+  describe(`Test contract origination of a simple contract through contract api using: ${rpc}`, () => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/wallet-simple-origination.spec.ts:5]: changed

File: integration-tests/wallet-tx-wait-2-confirmations.spec.ts [hunk 10]
```diff
@@ -2,14 +2,14 @@ import { CONFIGS } from "./config";
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
-  describe(`Test tx and waiting for 2 confirmations using: ${rpc}`, () => {
+  describe(`Test tx and waiting for 2 confirmations through wallet api using: ${rpc}`, () => {
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/wallet-tx-wait-2-confirmations.spec.ts]: changed

File: integration-tests/wallet-unit-as-param.spec.ts [hunk 11]
```diff
@@ -3,14 +3,14 @@ import { depositContractCode, depositContractStorage } from "./data/deposit_cont
 
 CONFIGS().forEach(({ lib, rpc, setup }) => {
   const Tezos = lib;
+  const test = require('jest-retries');
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/wallet-unit-as-param.spec.ts]: yes, removed

File: integration-tests/wallet-simple-origination.spec.ts (hunk 9 above)
[the remark this answers is not in the document]
Comment (roxaneletourneau) [reply integration-tests/wallet-simple-origination.spec.ts:5]: should be `through wallet API`, not contract

**Review by roxaneletourneau — CHANGES_REQUESTED** [review]

File: integration-tests/wallet-simple-origination.spec.ts (hunk 9 above)
[the remark this answers is not in the document]
Comment (roxaneletourneau) [reply integration-tests/wallet-simple-origination.spec.ts:5]: @michaelkernaghan you might have missed the last comment: should be through wallet API, not contract

**Review by roxaneletourneau — APPROVED** [review]

**Pull request merged**